synapse/storage/databases/main/devices.py

```diff
@@ -1638,19 +1638,22 @@ async def check_too_many_devices_for_user(self, user_id: str) -> List[str]:
             """
 
             rows = await self.db_pool.execute(
-                "check_too_many_devices_for_user_last_seen", None, sql, (user_id,)
+                "check_too_many_devices_for_user_last_seen",
+                None,
+                sql,
+                user_id,
```

Comment:
The problem here is that `execute` takes a `*args`, unlike `txn.execute` that takes a tuple